books/free-programming-books-langs.md

```diff
@@ -1335,6 +1335,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 #### Vue.js
 
 * [30 Days Of Vue](https://www.newline.co/30-days-of-vue) - Hassan Djirdeh (HTML; *email required for PDF*)
+* [Fullstack Vue: The Complete Guide to Vue.js and Friends](https://dl.ebooksworld.ir/motoman/fullstack-vue-book-r8.www.EBooksWorld.ir.pdf) - Hassan Djirdeh, Nate Murray, Ari Lerner (PDF)
```

Comment:
Is this resource accessible?

![image](https://user-images.githubusercontent.com/50091666/195395045-30c95b05-d13b-4a94-aa02-e13eec739895.png)
I am not able to access it

Reply:
Hello torre, I am able to access the link both on my computer and on my phone. When I click on the link, the book opens.
Do you have any VPNs turned on? Any proxies? Could you please try again on a different browser or device? 
[Here is the link](https://dl.ebooksworld.ir/motoman/fullstack-vue-book-r8.www.EBooksWorld.ir.pdf)